data-processing/entities/citations/extractor.py

```diff
@@ -52,6 +52,13 @@ def _extract_text(self, bibitem: TexSoup) -> str:
         for content in list(bibitem.contents)[1:]:
             if isinstance(content, TexNode) and content.string is not None:
                 text += content.string
+            elif (
```

Comment:
can add a comment here describing?  it seems clear to me this is capturing hrefs but not sure what the `>=2` and `isinstance, RArg` are doing

Reply:
ah i think i get it now, nvm

Reply:
Done!